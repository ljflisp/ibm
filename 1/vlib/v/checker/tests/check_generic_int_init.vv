fn test<T>() T {
	return T{}
}

fn main() {
	_ := test<int>()
}
