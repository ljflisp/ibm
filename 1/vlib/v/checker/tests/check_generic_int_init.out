vlib/v/checker/tests/check_generic_int_init.vv:2:9: error: type `int` is private
    1 | fn test<T>() T {
    2 |     return T{}
      |            ~~~
    3 | }
    4 |
